src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+    let totalCost = days * dailyRate;
```

Comment:
Prefer using 'const' over 'let' for 'totalCost' since it is not being reassigned, but rather its value is updated based on conditions.